b3/conf/b3.distribution.ini

```diff
@@ -50,6 +50,9 @@ delay: 0.33
 # Number of lines to process per second: set a lower value to consume less CPU ressources
 lines_per_second: 50
 
+permban_with_frozensand: no
```

Comment:
I think this should not be added to b3.distribution.ini at all, since that is a template file with options common to all the parsers.